Eastern-Research-Group/expert-query
# Feature/156 national downloads stream

## Related Issues:
* [EQ-156](https://jira.epa.gov/browse/EQ-156)

## Main Changes:
* Pulled the national downloads logic out of the extract from ords logic. This is to prepare for moving the ETL process over to loading data via pgdump files in S3.
* Added the ability to run the national downloads separately.

## Steps To Test:
1. Add the `STREAM_BATCH_SIZE: 2000` and `STREAM_HIGH_WATER_MARK: 10000` to your `.env.local` file in the etl
2. Run `npm run etl_national_downloads` 
3. Verify the national downloads were built and work.
4. Run `npm run etl_database`
5. Repeat step 2

## TODO:
- [ ] May be able to lower memory on the cloud.gov server. In my local tests, memory usage stayed below 300mb.
